src/main/java/gregtech/common/terminal/app/worldprospector/WorldProspectorARApp.java

```diff
@@ -81,12 +93,48 @@ public AbstractApplication initApp() {
             }
             RectButtonWidget buttonWidget = new RectButtonWidget(x + i * 24, y + 18, 18, 18, 1);
             addWidget(new PhantomSlotWidget(handlers[i], 0, x + i * 24, y) {
+
+                @Override
+                public List<IGhostIngredientHandler.Target<?>> getPhantomTargets(Object ingredient) {
+                    if (!(ingredient instanceof ItemStack)) {
+                        return Collections.emptyList();
+                    }
+                    Rectangle rectangle = toRectangleBox();
+                    return Lists.newArrayList(new IGhostIngredientHandler.Target<Object>() {
+                        @Nonnull
+                        @Override
+                        public Rectangle getArea() {
+                            return rectangle;
+                        }
+
+                        @Override
+                        public void accept(@Nonnull Object ingredient) {
+                            if (ingredient instanceof ItemStack) {
+                                int mouseButton = Mouse.getEventButton();
```

Comment:
Is `Mouse` safe to access here? Does client/server sidedness need to be somehow checked first?

Reply:
Dont think so, was copied from PhantomSlotWidget, just with 1 call added to the `accept` method